sdks/python/apache_beam/io/external/xlang_jdbcio_it_test.py

```diff
@@ -150,6 +154,18 @@ def test_xlang_jdbc_read(self):
       assert_that(
           result, equal_to([JdbcReadTestRow(i) for i in range(ROW_COUNT)]))
 
+  # Creating a container with testcontainers sometimes raises ReadTimeout
+  # error. In java there are 2 retries set by default.
+  def start_postgres_container(self, retries):
+    for i in range(retries):
+      try:
+        self.postgres = PostgresContainer('postgres:12.3')
+        self.postgres.start()
+        break
+      except:  # pylint: disable=bare-except
+        if i == retries - 1:
+          raise RuntimeError('Unable to initialize postgres container.')
```

Comment:
Log the error ?

Reply:
I added the log and reraised the exception cause.